test.rb

```diff
@@ -0,0 +1,26 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+aaa;
+
+not 1
+
+ l = lambda {|a| a }
+ l.(x)
+l[x]
+l.call(x)
+
+  def b(x)
+    
+  end 
+
+
+  ['x','b',"c"]
+    [x,a,b,c,d,e]
+    
+    foo (  x)
```

Comment:
__[development][rubocop]__
* Unnecessary spacing detected.
* Put one space between the method name and the first argument.
* Inconsistent indentation detected.
* Don't use parentheses around a method call.

<!-- Comment by [SideCI](https://sideci.com) -->